docs/source/user_guide/objectives.ipynb

```diff
@@ -21,7 +21,7 @@
    "source": [
     "## Core Objectives\n",
     "\n",
-    "Use the `get_objectives` method to get a list of which objectives are included with EvalML for each problem type:"
+    "Use the `get_core_objectives` method to get a list of which objectives are included with EvalML for each problem type:"
```

Comment:
Could we add an example of `get_default_primary_search_objective` somewhere too?

Reply:
Yes! The best place to add this would be in the section @angela97lin added in this open [PR](https://github.com/alteryx/evalml/pull/1222/files#diff-7a8ba774ebf1378e72f1793fb4603101R103) - so maybe that PR should be merged first.

Reply:
Waiting for tests to pass right now but should be merged in soon!